### DISCUSSION TITLE
Group, Groups

### DISCUSSION THREAD

**Discussion by tedztar** [NamelessMC/Nameless#2154] [Ideas]
**Please describe the feature in as much detail as possible**
So the server I dev for has 30+ groups currently, So it is a bit of a mess, so my suggestion has groups for the groups (Like categories for forums). This way you could sort your groups and apply perms to all of them quickly without going through each one on their own.

**Comment by samerton** [comment]
Interesting idea, just thinking an alternative to this would be the ability to copy permissions from a different group as you create it?

**Comment by tedztar** [comment]
I guess you could but I was more thinking they would act more as
inheritances then anything. Like discord perms when it comes to channels
and groups.

**Comment by puttydotexe** [comment]
One whole word:
**_Inheritance_**
